readme.md

```diff
@@ -81,7 +86,10 @@ Your Rails database has two tables. `students` has the columns `id` and `name`,
 Use ActiveRecord to create a new `pbj` sandwich and make it belong to the student named Geraldo.
 
 ```rb
-# Your answer...
+Sandwich.create({
+type: 'pbj',
+student_id: Student.find_by(name:'Geraldo')
+})
```

Comment:
Did you try running this? I believe `Student.find_by(name: 'Geraldo')` will return a whole record, rather than pointing directly to the ID.